### DISCUSSION TITLE
SM2135 Bulb wrong RGB order

### DISCUSSION THREAD

**Discussion by aproetz** [arendst/Tasmota#12670] [Q&A] [answered]
I finally got time to flash one of many of my Sunlit E27 10W RGBW bulbs.
Eventually I ended up opening the unit only to discover it is using the SM2135 chip.

Got a working template going (had to trace the pins).
{"NAME":"Sunlit RGBW E27","GPIO":[0,0,0,0,4032,0,0,0,0,0,4064,0,0,0],"FLAG":0,"BASE":18}

The only is now is that the colors are mixed up.
Blue = Outputs Red
Red = Outputs Green
Green = Outputs Blue.

How can I go about fixing this?  I dont have access to another linux device at the moment and would prefer not to tinker with the firmware.  

Thank you

**Comment by s-hadinger** [comment, marked as answer]
Look at SetOption37

**Reply by s-hadinger** [reply]
https://tasmota.github.io/docs/SetOption37/

**Reply by aproetz** [reply]
Thank you for the quick response.

SetOption37 31 did the trick!
